userspace/libsinsp/container.h

```diff
@@ -31,31 +36,38 @@ limitations under the License.
 
 #include "container_engine/container_engine.h"
 
+struct scap_dumper;
+class sinsp_evt;
+
 class sinsp_container_manager
```

Comment:
Please add doxy to some of this

Reply:
I added a few words though for decent docs we should probably split it into at least three pieces:
- the container table
- resolve_container() and friends
- utils like the health-check related stuff